### ISSUE TITLE
🐛 BUG: Parse error when CSS in markdown contains backslash

### ISSUE THREAD

**Issue by bramaudi** [snowpackjs/astro#1017]
### What package manager are you using?

pnpm

### What operating system are you using?

Linux

### Describe the Bug

Parse error when you write a CSS in markdown syntax higlight that contains backslash symbol "\"

<details>
<summary>Error logs</summary>
<pre><code>[parse-error] Legacy octal escape sequences cannot be used in template literals
[src/posts/2021/html-input-rating.md] Error: Unable to compile expression
[21:06:35] [snowpack] Build Result Error: There was a problem with a file build result.
[executing astro] Error: Parse error @:1:1
    at parse (/home/brama/Coding/blog/node_modules/.pnpm/es-module-lexer@0.3.26/node_modules/es-module-lexer/dist/lexer.cjs:1:402)
    at scanCodeImportsExports (/home/brama/Coding/blog/node_modules/.pnpm/snowpack@3.8.3/node_modules/snowpack/lib/cjs/rewrite-imports.js:21:29)
    at transformEsmImports (/home/brama/Coding/blog/node_modules/.pnpm/snowpack@3.8.3/node_modules/snowpack/lib/cjs/rewrite-imports.js:37:27)
    at Object.transformFileImports (/home/brama/Coding/blog/node_modules/.pnpm/snowpack@3.8.3/node_modules/snowpack/lib/cjs/rewrite-imports.js:108:16)
    at FileBuilder.resolveImports (/home/brama/Coding/blog/node_modules/.pnpm/snowpack@3.8.3/node_modules/snowpack/lib/cjs/build/file-builder.js:116:48)
    at async Object.loadUrl (/home/brama/Coding/blog/node_modules/.pnpm/snowpack@3.8.3/node_modules/snowpack/lib/cjs/commands/dev.js:640:39)
    at async load (/home/brama/Coding/blog/node_modules/.pnpm/snowpack@3.8.3/node_modules/snowpack/lib/cjs/commands/dev.js:217:28)
    at async /home/brama/Coding/blog/node_modules/.pnpm/snowpack@3.8.3/node_modules/snowpack/lib/cjs/ssr-loader/index.js:60:28
    at async /home/brama/Coding/blog/node_modules/.pnpm/snowpack@3.8.3/node_modules/snowpack/lib/cjs/ssr-loader/index.js:83:23
    at async initializeModule (/home/brama/Coding/blog/node_modules/.pnpm/snowpack@3.8.3/node_modules/snowpack/lib/cjs/ssr-loader/index.js:88:39)
    at async Object.importModule (/home/brama/Coding/blog/node_modules/.pnpm/snowpack@3.8.3/node_modules/snowpack/lib/cjs/ssr-loader/index.js:166:25)
    at async load (file:///home/brama/Coding/blog/node_modules/.pnpm/astro@0.18.8/node_modules/astro/dist/runtime.js:76:17)
    at async Server.<anonymous> (file:///home/brama/Coding/blog/node_modules/.pnpm/astro@0.18.8/node_modules/astro/dist/dev.js:21:20) {
  idx: 0
}</code></pre>
</details>

### Steps to Reproduce

Write something like this in markdown syntax highlight.

```css
.classes {
  content: "\2605";
}
```

Add one backslash again to escape will solve this but, that gonna break the CSS!

```css
.classes {
  content: "\\2605";
}
```
**EDIT**: Add error log

**Comment by jasikpark** [comment]
Maybe using a raw string rather than a template literal would work better wherever the code is erroring? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#raw_strings

**Comment by bramaudi** [comment]
Thanks for the reference. As referenced in template literal i'm trying to change unicode value to `\u2605` and its worked, but still in real css is not a valid css, looks like the problem is only when you try to escape a number.

This is okay i can just write unicode like ` content: "★";` but the error will be common pitfal in the future isn't it?

**Comment by natemoo-re** [comment]
Confirming that this is fixed in the `astro@next` preview! See [working reproduction](https://stackblitz.com/edit/github-x3dyfv-md6chb?file=src%2Fpages%2Findex.astro).

Will put this in our milestone to be closed when that release is stable.

**Comment by natemoo-re** [comment]
Following up on my previous comment, the team has decided to close out issues that have been confirmed as fixed by [astro@0.21.0-next.0](https://github.com/snowpackjs/astro/releases/tag/astro%400.21.0-next.0), [astro@0.21.0-next.1](https://github.com/snowpackjs/astro/releases/tag/astro%400.21.0-next.1), or [astro@0.21.0-next.2](https://github.com/snowpackjs/astro/releases/tag/astro%400.21.0-next.2). Our hope is that this will help the [v0.21 milestone](https://github.com/snowpackjs/astro/milestone/1) remain as actionable as possible.

To verify that this issue has been fixed, you may
- Open a new [`astro@next` project on StackBlitz](https://astro.new/minimal@next)
- Locally upgrade to the `astro@next` preview
    ```shell
    npm i astro@next
    # If using Framework renderers...
    npm i @astrojs/renderer-name@next
    ```

If you think we made a mistake, **please reopen this issue**!

If you run into any other problems with `astro@next`, please [open a new issue](https://github.com/snowpackjs/astro/issues/new?assignees=&labels=0.21&template=---01-bug-report.yml&title=%F0%9F%90%9B+BUG%3Aastro@next).
